packages/delegation-readmodel-writer/.env

```diff
@@ -0,0 +1,13 @@
+LOG_LEVEL=info
+
+KAFKA_CLIENT_ID="delegation"
+KAFKA_GROUP_ID="delegation-group"
+KAFKA_BROKERS="localhost:9092"
+KAFKA_DISABLE_AWS_IAM_AUTH="true"
+DELEGATION_TOPIC="event-store.delegation.events"
+READMODEL_DB_HOST="localhost"
+READMODEL_DB_NAME="readmodel"
+READMODEL_DB_USERNAME="root"
+READMODEL_DB_PASSWORD="example"
+READMODEL_DB_PORT=27017
+AWS_REGION="eu-central-1"
```

Comment:
See #1128 
```suggestion
region=eu-south-1
```

Reply:
already handled in this separate [PR](https://github.com/pagopa/interop-be-monorepo/pull/1159) that should be merged soon